packages/@ember/-internals/glimmer/tests/integration/components/link-to/routing-angle-test.js

```diff
@@ -1708,36 +1708,6 @@ moduleFor(
       assertNav({ prevented: true }, () => this.$('#self-link').click(), assert);
     }
 
-    async [`@test it should not transition if target is not equal to _self or empty`](assert) {
-      this.addTemplate(
-        'index',
-        `
-        <LinkTo id='about-link' @route='about' @replace={{true}} target='_blank'>
-          About
-        </LinkTo>
-        `
-      );
-
-      this.router.map(function () {
-        this.route('about');
-      });
-
-      await this.visit('/');
-
-      await this.click('#about-link');
-
-      expectDeprecation(() => {
-        let currentRouteName = this.applicationInstance
-          .lookup('controller:application')
```

Comment:
I don't think this test was specifically testing deprecated behavior, I think the deprecated API was only used during test of a legitimate behavior we want to retain.

I believe the right approach here is to refactor the test to use the router service instead of `currentRouteName` via the application controller, remove the `expectDeprecation(` code, and retain the test.

Can you dig into it?